### DISCUSSION TITLE
How to handle image on delete?

### DISCUSSION THREAD

**Discussion by lumpinif** [steven-tey/novel#389] [Q&A]
I have got image upload working as I expected with supabase as the backend. However when I removed the image, it stayed in the db obviously, but we should remove the image from the db for best practice. 

Does anyone have any idea how can we handle the image deletion both on the client and the backend?

**Comment by AKRking** [comment]
Have u figured out?cant find any resources regarding this important issue

**Reply by lumpinif** [reply]
Sorry I haven't got the time to handle the image deletion. As mentioned somewhere, the default behavior of caching the image allows the user to undo the deleted image back, which can be a problem if the image is removed from the DB.

So I guess the logic should be handled with both removing and undoing.

**Reply by AKRking** [reply]
currently following claude sonnet for the deleting process with a grain of salt. haven't implemented yet.

**Reply by lumpinif** [reply]
> currently following claude sonnet for the deleting process with a grain of salt. haven't implemented yet.

Claude is good. If it is working for you would you mind to share here?
